worlds/musedash/test/TestDifficultyRanges.py

```diff
@@ -68,7 +68,7 @@ def test_songs_have_difficulty(self) -> None:
 
             # umpopoff is a one time weird song. Its currently the only song in the game
             # with non-standard difficulties and also doesn't have 3 or more difficulties.
-            if song_name == 'umpopoff':
+            if song_name == "umpopoff" or song_name == "P E R O P E R O Brother Dance":
```

Comment:
```suggestion
            if song_name in ("umpopoff", "P E R O P E R O Brother Dance"):
```
Just in case they do it again.